src/commands/latency-latest.json

```diff
@@ -41,6 +41,14 @@
                     {
                         "type": "integer",
                         "description": "Max latency in milliseconds."
+                    },
+                    {
+                        "type": "integer",
+                        "description": "Sum of the latencies recorded in the time series for this event."
```

Comment:
Not sure if we will add history and since for the new fields. I see we did not do it in memory-stats

Reply:
We don't need it IMHO.